src/App.jsx

```diff
@@ -1,10 +1,121 @@
 import './App.css';
+import map from './assets/world_map.png';
+import axios from 'axios';
+import {useState} from 'react';
+import getRegionColors from './helpers/getRegionColors.js';
+import population from './helpers/population.js';
 
 function App() {
 
+    const [countries, setCountries] = useState([]);
+    const [searchCountry, setSearchCountry] = useState('');
+    const [countryInfo, setCountryInfo] = useState({});
```

Comment:
Als je onderscheidt wilt tussen of het al wél of niet is ingeladen, dan zou ik een default waarde als `undefined` nemen.
```suggestion
    const [countryInfo, setCountryInfo] = useState();
```
Je check verderop kan dan ook iets simpeler